sdk/aqueduct/integrations/dynamic_k8s_integration.py

```diff
@@ -42,11 +42,11 @@ def create(self) -> None:
             "Cluster is currently in %s status. It could take 12-15 minutes for the cluster to be ready..."
             % status.value
         )
-        globals.__GLOBAL_API_CLIENT__.edit_engine(
+        globals.__GLOBAL_API_CLIENT__.edit_dynamic_engine(
             action="create", integration_id=str(self._metadata.id)
         )
 
-    def delete(self) -> None:
+    def delete(self, force: bool = False) -> None:
```

Comment:
This is a user facing function, so let's please document it and especially note what `force` means.

Reply:
adding documentation to both delete and create.